### PR TITLE
Couple of improvements in the main README

### PR DESCRIPTION
**Pull request by osanseviero**

### CODE REVIEW

File: huggan/README.md [hunk 1]
```diff
@@ -63,35 +63,33 @@ source ~/<your-venv-name>/bin/activate
...
 ### Install Dependencies
 
-We've packaged up the example scripts here into a simple Python package. To install it, just clone it and pip install it locally. 
+We've packaged up the example scripts here into a simple Python package. To install it, just pip install it
 
 ```
-git clone https://github.com/huggingface/community-events.git
-cd community-events
-pip install .
+!pip install git+https://github.com/huggingface/community-events.git
```
Comment (nateraw) [inline huggan/README.md:69]:
```suggestion
pip install git+https://github.com/huggingface/community-events.git
```
I think we do this without the !

**Review by nateraw — APPROVED** [review]
LGTM. one nit

**Comment by nateraw** [comment]
@osanseviero so I merged this, but then I replaced the pip install in another commit to revert it back to what it was before. Folks should have this locally, as that's how we explain how to run the scripts, so it doesn't make as much sense to have them install it remotely.

**Comment by osanseviero** [comment]
Ah that makes sense, thanks!

**Pull request merged**